.github/workflows/psv_pipelines.yml

```diff
@@ -70,9 +49,12 @@ jobs:
     - name: Compile project with cmake and ccache
       run: gcc --version && ./scripts/linux/psv/build_psv.sh
       shell: bash
+    - name: Run unit and integration tests
+      run: ./scripts/linux/psv/test_psv.sh
+      shell: bash
 
-  psv-linux-gcc-build-no-cache:
-    name: PSV / Linux gcc 7.5 / OLP_SDK_ENABLE_DEFAULT_CACHE=OFF
+  psv-linux-20-04-gcc7-build-no-cache:
+    name: PSV.Linux.gcc7.OLP_SDK_ENABLE_DEFAULT_CACHE=OFF
```

Comment:
Just a notice: other names has ubuntu version in them.

Reply:
Fixed.